### ISSUE TITLE
Libraries don't seem to be available anymore

### ISSUE THREAD

**Issue by astrobellatrix** [cschreib/fastpp#22]
It seems the Berkeley website has finally conked out. They are not available anymore. Is there any way to get 
[ised_del.lr](http://pepper.astro.berkeley.edu/~mariska/FAST_Libraries/ised_del.lr.tar.gz)
[ised_del.hr](http://pepper.astro.berkeley.edu/~mariska/FAST_Libraries/ised_del.hr.tar.gz)
[ised_tru.hr](http://pepper.astro.berkeley.edu/~mariska/FAST_Libraries/ised_tru.hr.tar.gz)
[ised_exp.hr](http://pepper.astro.berkeley.edu/~mariska/FAST_Libraries/ised_exp.hr.tar.gz)
Thanks

**Comment by cschreib** [comment]
The only copies I have on my hard drive at the moment are `ised_del.hr` and `ised_exp.pr`. I may have a couple others lying around in older hard drives, that I could upload somewhere. But it's a lot of data to store, I don't know where this can be archived reliably for the long term.

The better alternative is to stop using these pre-compiled libraries, and switch to using [custom star formation histories](https://github.com/cschreib/fastpp?tab=readme-ov-file#custom-star-formation-histories). The docs show you how to reproduce the `del` libraries, and should be easy to adapt to `exp` and `tru` (let me know if not). This only relies on the SSP libraries, which are [still accessible](http://www.bruzual.org/bc03/Original_version_2003/).

**Comment by astrobellatrix** [comment]
I actually found them on an old FAST_v1.0 version on another computer, so no worries about me. I have also compiled the SSPs. Perhaps just update the readme file then. Thanks for answering so fast!

**Comment by cschreib** [comment]
Yes, the documentation needs updating. I'll leave this issue open until I get around to it. Thanks for letting me know!

**Comment by ryoalbert** [comment]
I have found the working link for pre-compiled fast libraries provided by James Aird: [FAST Libraries](http://cuillin.roe.ac.uk/~jaird/fast_libraries/). I think he updated the link on the original FAST github, but somehow, the updated link there is still not working.
